spec/support/utilities/iis_virtual_directory.rb

```diff
@@ -9,7 +9,7 @@ def create_vdir(vdir_name, site = 'foo', path = 'C:\inetpub\wwwroot')
 end
 
 def remove_vdir(vdir_name, site = 'foo', path = 'C:\inetpub\wwwroot')
-  command = format_powershell_iis_command("Remove-WebVirtualDirectory -Name #{vdir_name} -Site #{@site} -PhysicalPath #{@path}")
+  command = format_powershell_iis_command("Remove-Item -Path 'IIS:\\Sites\\#{@resource[:sitename]}\\#{@resource[:name]}' -Recurse -ErrorAction Stop")
```

Comment:
Acceptance tests failing because `@resource` is not a thing in this file. Need to translate to use the `vdir_name` @site` and `@path` objects instead.